packages/ui-components/src/components/radio-list-item/radio-list-item.helper.tsx

```diff
@@ -0,0 +1,29 @@
+import { JSX, h } from '@stencil/core';
+import { LINK_MATCHER_REGEX } from './radio-list-item.config';
+import { EAnchorTarget } from '../../types';
+
+export const buildDescription = (description?: string): JSX.Element[] => {
```

Comment:
Are we going forward with this approach? Seems very error prune.
Besides, what if want some part of the description to be bold? How would we achieve that?
Seems to me that a slots solution, despite requiring more work from the developer, would fit every use case we would need.

Reply:
Why don't we use something like this?

https://github.com/markedjs/marked
https://github.com/probablyup/markdown-to-jsx

Reply:
Thank you for the input, moving forward I'll isolate the parsing logic from this component, most likely using one of the aforementioned powerful libs!